server.go

```diff
@@ -100,16 +105,9 @@ func (s *Server) ListenAndServe() error {
 		log.Fatal(err)
 	}
 
-	secret, err := GetEnvironmentVariable("TIMETRACKER_SESSION")
-	if err != nil {
-		log.Fatal(err)
-	}
-	session := sessions.New([]byte(secret))
-	session.Lifetime = 12 * time.Hour
-
 	app := Application{
-		tasks:         &Env{Db: db},
 		templateCache: templateCache,
+		TaskStore:     &Env{Db: db},
```

Comment:
Okay. So it should be possible for users to create an `Application` using whatever kind of `Store`-satisfying object they want, shouldn't it? Can you figure out how to do that via a functional option?